pod/video/templates/videos/add_video.html

```diff
@@ -27,16 +27,17 @@
 
 
 {% block page_content %}
-<h3>{% trans "Choose an audio or video file to upload" %}</h3>
+<h3><label for="chunked_upload">{% trans "Choose an audio or video file to upload" %}</label></h3>
```

Comment:
pourquoi rajouter un label ?

Reply:
Une des règles en accessibilité, c'est que tout input doit avoir un label associé, de manière à ce qu'un utilisateur avec une synthèse vocale comprenne bien à quoi sert chaque input.
N'hésitez-pas à installer le ,plugin "Wave" dans votre navigateur pour faire un audit rapide d'accessibilité.

Reply:
Cependant, je ne vois pas de champ input qui précède ou succède ce H3 ?

Reply:
cherche l'id "chunked_upload".

Reply:
4 lignes en dessous.

Reply:
Oui je vois mais je me demande si le h3 n'est pas un titre de cette fonction et qu'il y a simplement un oubli de label ? @JoshuaBaubry tu peux nous eclairer ?

Reply:
Donc il faudra modifier ceci aussi Olivier.

Reply:
si j'ajoute juste un label avec le meme texte que le titre, ce sera redondant. Une idée de quoi mettre dans les 2 ?

Reply:
Que pensez-vous de conserver "Choose an audio or video file to upload" en label et mettre juste "File upload" en titre ?

Reply:
ca me va :) @JoshuaBaubry des suggestions ?

Reply:
Voila, j'ai fait un commit avec ca + compile des po

Reply:
ok parfait merci